### PR TITLE
{toolchain}[dummy/dummy] gomkl v2018b

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/8a370a98d443100552730554bcd36911 for a full test report.

**Comment by boegel** [comment]
@akesandgren Please include an `HPL` easyconfig using this toolchain, by means of test case?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3145.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/97d08c2dba6b3d91f3fb513e9742d699 for a full test report.

File: easybuild/easyconfigs/i/imkl/imkl-2018.3.222-gompi-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ applications that require maximum performance. Core math functions include
+ BLAS, LAPACK, ScaLAPACK, Sparse Solvers, Fast Fourier Transforms, Vector Math, and more."""
+
+toolchain = {'name': 'gompi', 'version': '2018b'}
+
+sources = ['l_mkl_%(version)s.tgz']
+checksums = ['108d59c0927e58ce8c314db6c2b48ee331c3798f7102725f425d6884eb6ed241']
+
+dontcreateinstalldir = 'True'
+
```
Comment (boegel) [inline easybuild/easyconfigs/i/imkl/imkl-2018.3.222-gompi-2018b.eb]: @akesandgren Any reason you dropped the `components = ['intel-mkl']` line here? Cfr. #6234

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/7e0db3989ef956510ce633804dc03f2c for a full test report.

File: easybuild/easyconfigs/i/imkl/imkl-2018.3.222-gompi-2018b.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+homepage = 'https://software.intel.com/en-us/intel-mkl/'
+description = """Intel Math Kernel Library is a library of highly optimized,
+ extensively threaded math routines for science, engineering, and financial
+ applications that require maximum performance. Core math functions include
+ BLAS, LAPACK, ScaLAPACK, Sparse Solvers, Fast Fourier Transforms, Vector Math, and more."""
+
+toolchain = {'name': 'gompi', 'version': '2018b'}
+
+sources = ['l_mkl_%(version)s.tgz']
```
Comment (boegel) [inline easybuild/easyconfigs/i/imkl/imkl-2018.3.222-gompi-2018b.eb]: @akesandgren `source_urls` is also missing, which we've started including in `imkl` easyconfigs recently...

```python
source_urls = ['http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/13005/']

File: easybuild/easyconfigs/i/imkl/imkl-2018.3.222-gompi-2018b.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/i/imkl/imkl-2018.3.222-gompi-2018b.eb]: Copied from wrong version.... fixing.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3112.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/9149083389ec974dd56239efcd17bda9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2424.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/db5638aa09d3c2758bf0863a4c98d662 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (3 easyconfigs in this PR)
hebbe-s1.int.private - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz, Python 2.7.14
See https://gist.github.com/2fbefea9ab5cdf7b0c28770e4e71c996 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**